lib/browser/api/net-log.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const {netLog, NetLog} = process.atomBinding('net_log')
+
+NetLog.prototype.stopLogging = function (callback) {
```

Comment:
Any reason why this check isn't performed in a similar way to `StartLogging` api using `mate::Arguments` ?

Reply:
Since the net log interface in Chromium for stop observing net logs takes `base::OnceClosure`, more work is needed to pass a callback function (which takes in a filepath) and wrap it in another closure to pass to Chromium.

```
NetLog.prototype.stopLogging(callback: (text: string) => void): void
NetLog.prototype._stopLogging(callback: () => void): void
```